packages/manager/apps/web/client/app/domain/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -223,7 +223,7 @@
                         data-placement="end"
                     >
                         <oui-action-menu-item
-                            data-on-click="$ctrl.goToWebhostingOrder()"
+                            data-ng-click="$ctrl.onWebhostingOrderClick()"
```

Comment:
Why do you stop using oui-action-menu-item on-click attribute ?

Reply:
Good question!

So, the raison is that `oui-action-menu-item` component is don't provide the on-click directive.
So i used Angular directive one.

Reply:
As discuss together yes it's a good observation.
Probably reading mistake with another component.

Fixed and thanks